src/hdmf/spec/namespace.py

```diff
@@ -272,9 +272,6 @@ def spec_namespace_cls(self):
             {'name': 'namespace', 'type': SpecNamespace, 'doc': 'the SpecNamespace object'})
     def add_namespace(self, **kwargs):
         """Add a namespace to this catalog"""
-        msg = ("NamespaceCatalog.add_namespace has been deprecated. "
-               "SpecNamespaces should be added with load_namespaces.")
-        warn(msg, DeprecationWarning)
```

Comment:
Why un-deprecate this function?
Why was it deprecated in the first place?

Reply:
I deprecated it because I thought you should only be able to add namespaces to a catalog using the load_namespaces functionality. That was more of a hunch than anything real. Eventually, I realized the utility in the add_namespaces functionality, so I'm undeprecating it.